### ISSUE TITLE
[BUG] KIBANA_HOST: unbound variable prevents deployment

### ISSUE THREAD

**Issue by kernel-sanders** [peasead/elastic-container#49]
#48 introduced a bug. Output of `./elastic-container start` after removing the `> /dev/null  2>&1` from the `set_fleet_values > /dev/null 2>&1` line:

```
Attempting to enable the Detection Engine and install prebuilt Detection Rules.

Kibana is up. Proceeding.

Detection engine enabled. Installing prepackaged rules.

Prepackaged rules installed!

Enabling detection rules

Successfully enabled Linux detection rules

Successfully enabled Windows detection rules

Successfully enabled MacOS detection rules

Waiting 40 seconds for Fleet Server setup.

Populating Fleet Settings.
./elastic-container.sh: line 143: KIBANA_HOST: unbound variable
```

`KIBANA_HOST` is only defined in the docker-compose for the fleet-server, but is used in the `elastic-container.sh` file.

**To Reproduce**
1. Run `./elastic-container.sh start` on a fresh system

**Expected behavior**
The script should not exit with status `1` on first run. It should setup the container in the same was as before #48.

 - OS: Debian 12

Some CI testing (build from scratch in a github actions runner, validate success) could be used to catch these types of issues before they make it to main.

**Comment by peasead** [comment]
It's defined in the `.env` file. Are you seeing this in other OS' or just Debian?

**Comment by peasead** [comment]
I take it back, not it's not.

**Comment by peasead** [comment]
@DefSecSentinel why would we not be defining this? It definitely was at some point. 

I wonder if this is causing any of the Fleet shakiness?

**Comment by kOld** [comment]
> `KIBANA_HOST` is only defined in the docker-compose for the fleet-server, but is used in the `elastic-container.sh` file.

My apologies, the last commit should have used `LOCAL_KBN_URL`. I've already created a pull request (#51) to address this. Could you please confirm if it works @DefSecSentinel?

**Comment by peasead** [comment]
I just ran this off of `main` and it deployed the rules properly.

While I agree the bug should be fixed, are you not seeing the rules enabled?

**Comment by DefSecSentinel** [comment]
I fixed this issue and validated the fix by standing up a fresh 8.15.0 stack on macOS with all Fleet settings properly populated and the expected output shown in the terminal.
